corehq/pillows/case_search.py

```diff
@@ -53,6 +53,7 @@
 )
 
 
+@memoized
```

Comment:
Does this never want to be cached between requests/processes, or does it not matter because its only used in pillows and those are very long running processes?

Reply:
This was my assumption, but I could be wrong?

Reply:
makes sense to me

Reply:
On second thought, does that present cache invalidation issues? Pillows run more or less for 24 hours (between each deploy). Does that mean if this feature is enabled for a domain, this function will continue to return wrong results until the next deploy, and if so, is that an issue?

Reply:
Good point. Updated here: https://github.com/dimagi/commcare-hq/pull/20715/commits/c3aab052ca230f7ccd3db44892c403e64ca89bba